docs/using.rst

```diff
@@ -297,7 +297,9 @@ need to issue this command in normal circumstances.
 
 ``--expand`` tells Certbot to update an existing certificate with a new
 certificate that contains all of the old domains and one or more additional
-new domains.
+new domains. With the ``--expand`` option, use the ``-d`` option to specify
+the new domain, and also use ``-d`` and re-type each existing domain in the certificate.
+For example: 
```

Comment:
Yes, eventually. Still waiting on answer to syntax question here: https://github.com/certbot/certbot/issues/3664

I pushed this commit just to keep things moving along, but my real focus yesterday was the Challenges section. The --expand stuff is not done yet, but I can finish it once I understand the syntax sufficiently (and then I'll add an example).